src/Bicep.Core/Semantics/Namespaces/AzNamespaceType.cs

```diff
@@ -26,14 +26,15 @@ public static class AzNamespaceType
     {
         public const string BuiltInName = "az";
         public const string GetSecretFunctionName = "getSecret";
-        private static string embeddedAzProviderVersion = typeof(AzTypeLoader).Assembly.GetCustomAttribute<AssemblyFileVersionAttribute>()?.Version ?? throw new UnreachableException("The 'Azure.Bicep.Types.Az' assembly should always have a file version attribute.");
+        private static string EmbeddedAzProviderVersion = typeof(AzTypeLoader).Assembly.GetCustomAttribute<AssemblyFileVersionAttribute>()?.Version
+            ?? throw new UnreachableException("The 'Azure.Bicep.Types.Az' assembly should always have a file version attribute.");
 
         public static NamespaceSettings Settings { get; } = new(
             IsSingleton: true,
             BicepProviderName: BuiltInName,
             ConfigurationType: null,
             ArmTemplateProviderName: "AzureResourceManager",
-            ArmTemplateProviderVersion: new Version(embeddedAzProviderVersion).ToString(3));
+            ArmTemplateProviderVersion: new Version(EmbeddedAzProviderVersion).ToString(3));
 
         private static FunctionOverload.ResultBuilderDelegate AddDiagnosticsAndReturnResult(TypeSymbol returnType, DiagnosticBuilder.DiagnosticBuilderDelegate writeDiagnostic)
         {
```

Comment:
This is not introduced by this PR, but `functionCall.Arguments.ToImmutableArray()` below can be simplied to `functionCall.Arguments` since the property is already an immutable array.